curriculum/challenges/english/20-upcoming-python/learn-list-comprehension-by-building-a-case-converter-program/657f025ec86c3d7c4177b6be.md

```diff
@@ -0,0 +1,40 @@
+---
+id: 657f025ec86c3d7c4177b6be
+title: Step 11
+challengeType: 20
+dashedName: step-11
+---
+
+# --description--
+
+Since the function is now complete, put it to use inside another function. Create a new function called `main` on the same level as the `convert_to_snake_case` function.
```

Comment:
```suggestion
Since the function is now complete, put it to use inside another function. Create a new function called `main()` on the same level as the `convert_to_snake_case()` function.
```